js/main.js

```diff
@@ -1,2 +1,40 @@
 /* global data */
 /* exported data */
+var $imgUrl = document.querySelector('input[id="photoUrl"]');
+var $formSubmit = document.querySelector('form');
+var $input = document.querySelectorAll('input');
+var $textArea = document.querySelector('textarea');
+var $setImg = document.querySelector('img');
+
+// Listening for input changes for IMAGE
+$imgUrl.addEventListener('input', changeImg);
+function changeImg(event) {
+  $setImg.setAttribute('src', event.target.value);
+}
+
+// Listening for submit event
+$formSubmit.addEventListener('submit', logForm);
+function logForm(event) {
+  event.preventDefault();
+  var formObject = {
+    title: $input[0].value,
+    photoUrl: $input[1].value,
```

Comment:
What if the order of your inputs changes? This is not a good path to follow, you should use the named properties on the object store in the `elements` property on your `form` element